wiki/Introduction-to-Creating-a-Job.md

```diff
@@ -0,0 +1,1818 @@
+Fri Jul  5 09:33:02 2024	----------------------------------------------
+Fri Jul  5 09:33:02 2024	Running command /private/var/folders/93/p7mk60kn69ggyc87dfbwtht80000gr/T/OpenJD/sample_session0bd5l1au/tmphe6bjl3c.sh
+Fri Jul  5 09:33:02 2024	Command started as pid: 85706
+Fri Jul  5 09:33:02 2024	Output:
+Fri Jul  5 09:33:03 2024	Blender 4.1.1 (hash e1743a0317bc built 2024-04-16 00:06:22)
+Fri Jul  5 09:33:03 2024	Error: Cannot read file "/private/var/folders/93/p7mk60kn69ggyc87dfbwtht80000gr/T/OpenJD/sample_session0bd5l1au/3d/pavillon_barcelone_v1.2.blend": No such file or directory
+Fri Jul  5 09:33:03 2024	
+Fri Jul  5 09:33:03 2024	Blender quit
+...
+```
+
+Uh-oh, there's an error! Blender couldn't find the input file. This is because all actions in a Session are always run with their
+current working directory being the Session's temporary working directory
+(`/private/var/folders/93/p7mk60kn69ggyc87dfbwtht80000gr/T/OpenJD/sample_session0bd5l1au` in this case), the scene file argument to your
+shell script was given as a relative path to the current working directory, and the scene file isn't in the Session's temporary
+working directory (it's actually in `/Users/myusername/blender_demo/3d/pavillon_barcelone_v1.2.blend` on this workstation).
+So, let's fix that up by changing the arguments to the command that the Task runs (note that we'll also have the same issue with
+the `output_frames` directory, so we'll preemptively fix that as well):
+
+```yaml
+...
+        onRun:
+          # Note: Task.File.Render refers to the file location on disk where the
+          #  contents of the embedded file named "Render" are materialized.
+          command: "{{Task.File.Render}}"
+          args:
+            - /Users/myusername/blender_demo/3d/pavillon_barcelone_v1.2.blend
+            - /Users/myusername/blender_demo/output_frames
+            - "1"
+            - "2"
+...
+```
+
+Then check the template syntax and try to run it again:
+
+```bash
+% openjd check job.template.yaml            
+Template at 'job.template.yaml' passes validation checks.
+% openjd run --step BlenderRender job.template.yaml
+Fri Jul  5 09:42:01 2024	
+Fri Jul  5 09:42:01 2024	==============================================
+Fri Jul  5 09:42:01 2024	--------- Running Task
+Fri Jul  5 09:42:01 2024	==============================================
+...
+Fri Jul  5 09:42:18 2024	Open Job Description CLI: All actions completed successfully!
+Fri Jul  5 09:42:18 2024	Open Job Description CLI: Local Session ended! Now cleaning up Session resources.
+
+--- Results of local session ---
+
+Session ended successfully
+
+Job: DemoJob
+Step: BlenderRender
+Duration: 17.484004959 seconds
+Tasks run: 1
+```
+
+Success! Also verify that the output are where we expect them to be:
+
+```bash
+% ls /Users/myusername/blender_demo/output_frames 
+frame-001.png	frame-002.png
+```
+
+Before we continue adding functionality to your Job we're going to have you add a timeout to the Task. The timeout
```

Comment:
Bug to fix, ideally before this merges: https://github.com/OpenJobDescription/openjd-cli/issues/96